install/empty_data.php

```diff
@@ -205,6 +205,7 @@ public function getEmptyData(): array
             'is_location_autoclean' => '0',
             'state_autoclean_mode' => '0',
             'use_flat_dropdowntree' => '0',
+            'use_flat_dropdowntree_on_display' => '1',
```

Comment:
IMO the current behavior should be kept as the the default one.

Reply:
Current behavior is to diplay completename (so true =) )

Reply:
my bad, fixed